language-design.md

```diff
@@ -80,4 +128,9 @@ you do so? If not, why not?
 
 **Response**
 
+Absolutely not. I can see why it is extremely difficult to write code in a natural language, we all have different ways 
+of saying the same thing. To try and force somebody to speak in a way that is different than they usually do is to rewrite
+the rules that thier language consists of. I think that it is much easier to learn a simple language and build your own terms within it.
```

Comment:
I definitely agree with this. The part of the Cook quote I liked most was "It is easy to read AppleScript, but quite hard to write it." This really emphasizes your point of how natural languages may be easier for novices to understand, but much more difficult to actually use as a programmer.